app/javascript/components/patient/household/actions/MoveToHousehold.js

```diff
@@ -37,18 +37,7 @@ class MoveToHousehold extends React.Component {
         tz_offset: new Date().getTimezoneOffset(),
         exclude_patient_id: this.props.patient.id,
         // This query should always filter out records that are not self-reporters
-        filter: [
-          {
-            dateOption: null,
-            filterOption: {
-              description: 'Monitorees that are a Head of Household or self-reporter',
-              name: 'hoh',
-              title: 'Daily Reporters (Boolean)',
-              type: 'boolean',
-            },
-            value: true,
-          },
-        ],
+        filter: [{ name: 'hoh', type: 'boolean', value: true }],
```

Comment:
For the Move to Household modal, its table should display the same results if I were to apply the "Daily Reporters" filter to the Global Dashboard's All Monitoree's line list, correct? When I tested the results for these scenarios, the results were very different (27 records were returned for the Move to Household table, and 1027 records were returned for the Main Dashboard when the "Daily Reporteds" filter is applied).

Reply:
rebased